tests/e2e/files/rulebooks/test_disabled_rules.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Ruleset 1
+  hosts: all
+  sources:
+    - generic:
+        event_delay: 0.5
```

Comment:
This should be a variable

Reply:
The default event delay is too high so we won't use that. Do you mean to specify the value in the test, or add a new key in default.yml?